### ISSUE TITLE
`react-imask` TypeScript error: Property 'mask' does not exist on type 'IntrinsicAttributes & { children?: ReactNode; }'.

### ISSUE THREAD

**Issue by LingVuDev** [uNmAnNeR/imaskjs#581]
**Describe the bug**
I started using `react-imask` package with TypeScript and I got an error on the example:

```ts
import { IMaskInput } from 'react-imask';

<IMaskInput
  mask={Number}
  radix="."
  value="123"
  unmask={true} // true|false|'typed'
  inputRef={el => this.input = el}  // access to nested input
  // DO NOT USE onChange TO HANDLE CHANGES!
  // USE onAccept INSTEAD
  onAccept={
    // depending on prop above first argument is
    // `value` if `unmask=false`,
    // `unmaskedValue` if `unmask=true`,
    // `typedValue` if `unmask='typed'`
    (value, mask) => console.log(value)
  }
  // ...and more mask props in a guide

  // input props also available
  placeholder='Enter number here'
/>
```

```
TypeScript error in ...:
Type '{ mask: NumberConstructor; radix: string; value: string; unmask: boolean; inputRef: (el: any) => any; onAccept: (value: any) => void; placeholder: string; }' is not assignable to type 'IntrinsicAttributes & { children?: ReactNode; }'.
  Property 'mask' does not exist on type 'IntrinsicAttributes & { children?: ReactNode; }'.  TS2322

    113 |         <IMaskInput
  > 114 |           mask={Number}
        |           ^
    115 |           radix="."
    116 |           value="123"
    117 |           unmask={true} // true|false|'typed'
```

**To Reproduce**

[![Edit gallant-germain-hrk8p](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/gallant-germain-hrk8p?fontsize=14&hidenavigation=1&theme=dark)

**Expected behavior**

**Environment:**
 - OS: MacOS
 - Browser Chrome
 - Version 95.0.4638.69
 - IMask version 6.2.2
 - Framework/plugin version if used: react

**Comment by keemor** [comment]
Take look here https://github.com/uNmAnNeR/imaskjs/issues/554

**Comment by uNmAnNeR** [comment]
fixed in 83f8ce8
